.github/workflows/python/lint_check.yaml

```diff
@@ -0,0 +1,29 @@
+name: Linting
+
+on:
+  workflow_call:
+
+jobs:
+  linting:
+    name: Linting
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
Not so sure about hardcoding this. I imagine this may differ among our python projects. That being said, I also don't know how to do it better...

Reply:
well, for our 3 cases this works, so I believe we can live with that at this moment

Reply:
I approve this message.